src/http/api/account.ts

```diff
@@ -256,6 +256,7 @@ export function createGetAccountPostsHandler(
             ctx.get('site'),
         );
         const { results, nextCursor } = await postService.getPostsByAccount(
+            account.id,
```

Comment:
For now, we're passing `account.id` as `defaultAccount` because they are the same in this context, but this will change in my next PR.